example/test_problem/Hydro/GrackleTest/yt_script/plot_grackle_analytical_cooling_rate.py

```
@@ -0,0 +1,239 @@
+    print(f"Metal-Free Gas Rates (z={Z_NOW}, nH={NH_NOW:.2e}, T={T_NOW:.2e}K)")
+    print_one_point(*calculate_one_point())
+
+
+def calculate_one_point(grackle_path=GRACKLE_PATH, z_now=Z_NOW, nH_now=NH_NOW, T_now=T_NOW):
+    cooling_rate, heating_rate, mmw = get_grackle_rates_and_mmw(grackle_path, z_target=z_now, nh_target=nH_now, t_target=T_now)
+    net_rate   = abs(heating_rate[0,0] - cooling_rate[0,0])
+    t_cool_Myr = calculate_cooling_time(nH_now, T_now, net_rate, mmw[0,0])
+    return cooling_rate, heating_rate, net_rate, mmw, t_cool_Myr
+
+
+def print_one_point(cooling_rate, heating_rate, net_rate, mmw, t_cool_Myr):
```

Ruff (0.15.5)

[warning] Unused function argument: `net_rate`

(ARG001)